client/my-sites/current-site/sidebar-banner/style.scss

```diff
@@ -1,30 +1,38 @@
-a.sidebar-banner {
-	display: flex;
-	background-color: $alert-green;
-	color: $white;
+.sidebar-banner {
 	font-size: 12px;
+	transition: all 150ms ease-out;
 
-	:visited {
-		color: $white;
+	:hover {
+		background-color: lighten( $alert-green, 5% );
 	}
 
-	@include breakpoint( "<660px" ) {
-		padding: 0 24px;
-	}
+	a {
+		background-color: $alert-green;
```

Comment:
I noticed the transition doesn't seem to work (the color change is instant) so I checked the code and it seems the color gets applied to the `.sidebar-banner:hover` instead of `.sidebar-banner a` (odd, why does it work?). I think moving the transition and hover state to the `a` tag should fix this.

Reply:
Yep - i followed the same approach as for the menu items which seem to have hover on a <li>.

The problem with putting the hover on the <a> was that only one of the three spans would change color (whichever section you were hovering over) and not the entire <a> :/